_maps/map_files/PubbyStation/PubbyStation.dmm

```diff
@@ -9906,8 +9901,13 @@
 /turf/open/floor/plating,
 /area/bridge)
 "aue" = (
+/obj/structure/closet/secure_closet/freezer/money,
+/obj/item/weapon/reagent_containers/food/drinks/bottle/vodka/badminka,
+/obj/item/weapon/reagent_containers/food/drinks/drinkingglass/shotglass,
+/obj/item/weapon/reagent_containers/food/drinks/drinkingglass/shotglass,
+/obj/item/clothing/head/bearpelt,
+/obj/item/weapon/skub,
 /obj/machinery/atmospherics/pipe/simple/supply/hidden,
-/obj/structure/statue/gold/hop,
```

Comment:
Not really anywhere good to put this, all the wall space is taken up, it was either this or the skub